tycho-plugins/repository-utils/pom.xml

```diff
@@ -71,6 +71,11 @@
 			<artifactId>git-commit-id-plugin</artifactId>
 			<version>2.1.13</version>
 		</dependency>
+		<dependency>
+		    <groupId>org.tukaani</groupId>
```

Comment:
org.apache.commons.compress.compressors.xz depends on it:

"The classes in this package are wrappers around org.tukaani.xz.XZInputStream and org.tukaani.xz.XZOutputStream provided by the public domain XZ for Java library." - https://commons.apache.org/proper/commons-compress/apidocs/org/apache/commons/compress/compressors/xz/package-summary.html